api/src/pcapi/scripts/subscription/ubble/update_pending_ubble_applications.py

```diff
@@ -0,0 +1,34 @@
+"""
+Sometimes the ubble webhook fails. This is a problem because the application is still pending.
```

Comment:
Sometimes the ubble webhook fails. -> Sometimes the ubble webhook does not reach our api.